client/my-sites/post-type-list/post-type-post-author/index.jsx

```diff
@@ -1,38 +1,43 @@
 /**
  * External dependencies
  */
-import React, { PropTypes } from 'react';
+import React from 'react';
+import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 import { get } from 'lodash';
 import { localize } from 'i18n-calypso';
-import Gridicon from 'gridicons';
 
 /**
  * Internal dependencies
  */
+import { isEnabled } from 'config';
 import { getPost } from 'state/posts/selectors';
 import { isSingleUserSite } from 'state/sites/selectors';
+import { areAllSitesSingleUser } from 'state/selectors';
+import { getSelectedSiteId } from 'state/ui/selectors';
 
-function PostTypePostAuthor( { translate, singleUserSite, name } ) {
-	if ( ! name || singleUserSite ) {
+function PostTypePostAuthor( { singleUserSite, isAllSitesModeSelected, allSitesSingleUser, name } ) {
+	if ( ! isEnabled( 'posts/post-type-list' ) ||
+		! name ||
+		( isAllSitesModeSelected && allSitesSingleUser ) ||
+		( ! isAllSitesModeSelected && singleUserSite ) ) {
```

Comment:
As noted above, I think we can avoid depending on `isEnabled`, `singleUserSite`, `isAllSitesModeSelected`, and `allSitesSingleUser` here, by allowing the parent component to decide whether to render this component.

`if ( ! name )` - is this condition possible / likely / easy to reproduce?  When that happens, should this component return a placeholder instead?

Reply:
I just pushed a commit to allow the parent to decide about the visibility of the child component.

Reply:
I haven't been able to reproduce `if ( ! name )` but that condition was there before so there may be circumstances where it is necessary. I don't think we need a placeholder (see my comment about site name.)